workflows/long-read-mngs/run.wdl

```diff
@@ -211,12 +263,14 @@ task RunReadsToContigs {
 
         # convert non-contigs.fastq file to .fasta file
         seqtk seq -a sample.non_contigs.fastq > sample.non_contigs.fasta
+        cat sample.reads_to_contigs.sam | grep -v "^@" | cut -f1,3,10 > reads_to_contigs.tsv
```

Comment:
maybe adding `--secondary=no` to minimap2 might be good to keep multiple hits from the output.

Reply:
actually I don't think this works